apps/organization/src/organization.service.ts

```diff
@@ -195,7 +196,7 @@ export class OrganizationService {
         throw new NotFoundException(ResponseMessages.organisation.error.profileNotFound);
       }
 
-      const credentials = await this.organizationRepository.getCredDefByOrg(organizationDetails['id']);
+      const credentials = await this.organizationRepository.getCredDefByOrg(organizationDetails.id);
```

Comment:
Please use meaningful names for variables. In this case, instead of "credentials" it should be "credDefs".